core/storage/email/gae_models.py

```diff
@@ -194,18 +193,17 @@ def _generate_hash(cls, recipient_id, email_subject, email_body):
     @classmethod
     def check_duplicate_message(cls, recipient_id, email_subject, email_body):
         """Check for a given recipient_id, email_subject and cleaned
-        email_body, whether a similar message has been sent in the last
-        DUPLICATE_EMAIL_INTERVAL_MINS.
+            email_body, whether a similar message has been sent in the last
```

Comment:
Don't indent these two lines. It's not needed here because this is not a list of things.